src/Raven.Server/Web/RequestHandler.cs

```diff
@@ -54,8 +54,14 @@ public virtual void Init(RequestHandlerContext context)
 
         protected Stream RequestFormStream(string itemName)
         {
-            //TODO Handle exceptions
-            return new MemoryStream(Encoding.UTF8.GetBytes(HttpContext.Request.Form[itemName]));
+            try
```

Comment:
Why are you doing it like this?
Why pass form variables anyway? Just have then sent as the request body and that is it.